README.rst

```diff
@@ -77,57 +79,40 @@ Windows
 Using the API
 -------------
 
+Connecting Cloud Logging to `Python's standard logging library`_
+
 .. code:: python
 
-    from google.cloud import logging_v2
-    client = logging_v2.LoggingServiceV2Client()
-
-    resource = {
-        "type": "global",
-        "labels": {
-            "project_id": "[PROJECT_ID]"
-        }
-    }
-
-    """
-    Log entries can be either LogEntry or dict.
-    You can describe the same data in the following format:
-
-    e = {
-        "log_name": "projects/[PROJECT_ID]/logs/test-logging",
-        "resource": resource,
-        "text_payload": "this is a log statement",
-    }
-    """
-    e = logging_v2.types.LogEntry(
-        log_name="projects/[PROJECT_ID]/logs/test-logging", # optional
-        resource=resource, # optional
-        text_payload="this is a log statement")
-
-    entries = [e]
-    response = client.write_log_entries(entries)
+    # Imports the Cloud Logging client library
+    import google.cloud.logging
```

Comment:
I prefer not to have any codeblocks in the README. There's no good way to test them and it seems like a matter of time until they fall out of date.

Reply:
Good point. My intention was to remove the outdated samples, but I'm fine with removing them altogether. Done